Refresh.GameServer/Endpoints/Game/DataTypes/Response/GameLevelResponse.cs

```diff
@@ -239,4 +240,62 @@ public static GameLevelResponse FromHash(string hash, DataContext dataContext)
     }
 
     public static IEnumerable<GameLevelResponse> FromOldList(IEnumerable<GameLevel> oldList, DataContext dataContext) => oldList.Select(old => FromOld(old, dataContext)).ToList()!;
+    
+    public static GameLevelResponse? FromOld(GamePlaylist? old, DataContext dataContext)
+    {
+        if (old == null)
+            return null;
+        
+        return new GameLevelResponse
+        {
+            LevelId = old.PlaylistId,
+            IsAdventure = false,
+            Title = old.Name,
+            IconHash = old.Icon,
+            Description = old.Description,
+            Location = new GameLocation(old.LocationX,
+                old.LocationY),
+            // Playlists are only ever serialized like this in LBP1-like builds
+            GameVersion = TokenGame.LittleBigPlanet1.ToSerializedGame(),
```

Comment:
Why not just set directly to `0`?
```suggestion
            // Playlists are only ever serialized like this in LBP1-like builds
            GameVersion = 0 // TokenGame.LittleBigPlanet1
```

Reply:
No direct reason other than its more visually explicit and naming stays synced if we ever rename `TokenGame` or `LittleBigPlanet1`. This will get promoted to a JIT constant, so theres no runtime performance hit to this.